### PR TITLE
feat: Add Replication TestPlan

### PR DESCRIPTION
**Pull request by mykola-vrmchk**
### Motivation / Background

This Pull Request ...

### CODE REVIEW

File: packages/core/echo/echo-db/src/testing/echo-test-builder.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export class EchoTestPeer extends Resource {
     const db = client.constructDatabase({ spaceKey });
     await db.setSpaceRoot(rootUrl);
     await db.open();
+    this._ctx.onDispose(() => db.close());
```
Comment (dmaretskyi) [inline packages/core/echo/echo-db/src/testing/echo-test-builder.ts:121]: EchoClient already closes all child databases on close. Is there something broken that's caused you to add this?

File: packages/gravity/blade-runner/src/redis/defaults.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export const DEFAULT_WEBSOCKET_ADDRESS = `ws://${DEFAULT_WEBSOCKET.host}:${DEFAULT_WEBSOCKET.port}`;
+
+export const DEFAULT_REDIS_TCP_CONNECTION: NetConnectOpts = {
+  host: 'localhost',
+  port: REDIS_PORT,
+  family: 4,
+};
+
+export const DEFAULT_REDIS_OPTIONS = !isNode()
```
Comment (dmaretskyi) [inline packages/gravity/blade-runner/src/redis/defaults.ts:28]:
```suggestion
export const DEFAULT_REDIS_OPTIONS: RedisOptions = !isNode()
```

File: packages/gravity/blade-runner/src/redis/redis.test.ts [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
+import { REDIS_PORT } from './defaults';
+import { createRedisReadableStream, createRedisRpcPort, createRedisWritableStream } from './util';
 
 /**
  * NOTE(mykola): This test is disabled because it requires a running Redis server.
  *               `redis-server --port 6378` to start a Redis server on port 6378.
  * TODO(mykola): Mock Redis server.
  */
-describe.skip('Redis', () => {
+describe('Redis', () => {
```
Comment (dmaretskyi) [inline packages/gravity/blade-runner/src/redis/redis.test.ts:24]: forgot to skip, or do you start redis in the test?

File: packages/core/echo/echo-db/src/testing/echo-test-builder.ts (hunk 1 above)
Comment (mykola-vrmchk) [reply packages/core/echo/echo-db/src/testing/echo-test-builder.ts:121]: Ok, I did not know this

File: packages/gravity/blade-runner/src/redis/redis.test.ts (hunk 3 above)
Comment (mykola-vrmchk) [reply packages/gravity/blade-runner/src/redis/redis.test.ts:24]: Just a mistake

**Pull request merged**